src/graph.js

```diff
@@ -47,18 +49,20 @@ export default class Graph {
     if (!this._history) return;
     this._updateEndTime();
 
-    const coords = this._history.reduce((res, item) => this._reducer(res, item), []);
+    const histGroups = this._history.reduce((res, item) => this._reducer(res, item), []);
 
     // drop potential out of bound entry's except one
-    if (coords[0] && coords[0].length) coords[0] = [coords[0][coords[0].length - 1]];
+    if (histGroups[0] && histGroups[0].length) {
+      histGroups[0] = [histGroups[0][histGroups[0].length - 1]];
+    }
 
     // extend length to fill missing history
     const requiredNumOfPoints = Math.ceil(this.hours * this.points);
-    coords.length = requiredNumOfPoints;
+    histGroups.length = requiredNumOfPoints;
 
-    this.coords = this._calcPoints(coords);
-    this.min = Math.min(...this.coords.map(item => Number(item[V])));
-    this.max = Math.max(...this.coords.map(item => Number(item[V])));
+    this.histGroups = this._calcPoints(histGroups);
```

Comment:
You have edited little bit too much now ;) The local var name which I have asked to change was confusing because it wasn't containing graph coords and the name was exactly the same as the class property (so the impression was that the type of the value was the same - which was not true). 
Class property `this.histGroups` doesn't contain grouped history values (as name suggests) but actually calculated graph points/coords. Please leave here the old name.